DIPs/DIP1xxx-JM.md

```diff
@@ -0,0 +1,151 @@
+# Delegateable Functions
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | 1xxx                                                            |
+| RC#             | 0                                                               |
+| Author:         | Jonathan Marler - johnnymarler@gmail.com                        |
+| Implementation: | None                                                            |
+| Status:         | Will be set by the DIP manager (e.g. "Approved" or "Rejected")  |
+
+## Abstract
+
+UFCS allows functions to be called like member functions, however, they are limited in that they cannot be converted to delegates. This is because in most cases they are not ABI-compatible. It is therefore proposed to add sematics to define functions that are ABI-compatible with delegates, called "delegateable functions" and utilize UFCS to retreive delegates for these functions with an object of compatible type.
```

Comment:
"It is therefore proposed to add sematics to define functions"
"n" is missing in "semantics".